src/hyperlink/test/test_url.py

```diff
@@ -10,9 +10,7 @@
 
 from .common import HyperlinkTestCase
 from .. import URL, URLParseError
-# automatically import the py27 windows implementation when appropriate
-from .. import _url
```

Comment:
`_url` is not used, and tests are passing, but the comment makes me nervous here.
@mahmoud: perhaps you could double-check this bit…

Reply:
The comment and the import are from almost a year apart according to blames. The comment relates to inet_pton testing, and really doesn't add much. If tests pass we're fine :)